pyquil/quilbase.py

```diff
@@ -909,13 +921,28 @@ def __init__(self, qubitsA, frameA, qubitsB, frameB):
         self.frameA = frameA
         self.frameB = frameB
 
+    def out(self):
+        ret = "SWAP-PHASES"
+        for q in self.qubitsA:
+            ret += f" {q}"
+        ret += f" {self.frameA}"
+        for q in self.qubitsB:
+            ret += f" {q}"
+        return ret + f" {self.frameB}"
```

Comment:
Just an idea, but it looks like most of these `out` methods that print "single line" Quil could be simplified with a helper function like

```python
def _join_strings(*args):
    return ' '.join(map(str, args))
```

Then the `out` methods could be something like

```python
def out(self):
    return _join_strings("SWAP-PHASES", *self.qubitsA, self.frameA, *self.qubitsB, self.frameB)
```

My Python is rusty though, so maybe `for` loops and f-strings are better here for some reason.